src/lib/compiler.ts

```diff
@@ -1428,7 +1471,51 @@ export default class Compiler {
     this.lastSourceCommentRange = [node.getStart(), node.getEnd()];
   }
 
-  prettyTeal() {
+  appSpec(): object {
+    const approval = Buffer.from(this.prettyTeal()).toString('base64');
+    const clear = Buffer.from('#pragma version 8\nint 1; return').toString('base64');
```

Comment:
what do here? a specific method in the contract for clear state program handler?